pkg/scheduler/framework/types.go

```diff
@@ -39,6 +39,61 @@ import (
 
 var generation int64
 
+// ActionType is an integer to represent one type of resource change.
+// Different ActionTypes can be bit-wised to compose new semantics.
+type ActionType int64
+
+// Constants for ActionTypes.
+const (
+	Add    ActionType = 1 << iota // 1
+	Delete                        // 10
+	// UpdateNodeXYZ is only applicable for Node events.
+	UpdateNodeUnschedulable // 100
```

Comment:
Aren't we deprecating this soon?
Maybe we can just use NodeTaint for the plugin that need it today.

Reply:
Yes, basically I'd not expect anyone to use it.

I just put it for completeness and how about removing it once spec.unschedulable is officially deprecated?

Reply:
Better not, so that out-of-tree plugins don't use it

Reply:
SG. Updated in the incremental commit.